JS/Lesson34/gulcin/recipe-app/src/App.js

```diff
@@ -12,12 +13,9 @@ export const App = () => {
 			.then((data) => data.json())
 			.then((recipeData) => 
 			setRecipes(recipeData.meals));
-	}, []);
+	}, [])
 
 	return (
-		<div className={`app ${bgColor}`}>
-      <Search setRecipes={setRecipes}/>
-			<ul className='recipes__list'>{recipes.map((recipe => <li key={recipe.idMeal} >{recipe.strMeal}</li>))}</ul>
-		</div>
+			<AppRouter recipes={recipes} setSearchError={setSearchError} setRecipes={setRecipes} searchError={searchError}/>
```

Comment:
No need to change in this PR but you're missing Context for recipes. Recipe data should be updated and share using Context or Redux. Make sure to update it once we learn Redux.